src/components/dialogs/parameters/sensi/table-row.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { TableCell, TableRow } from '@mui/material';
+import Tooltip from '@mui/material/Tooltip';
+import IconButton from '@mui/material/IconButton';
+import { DeleteForeverOutlined } from '@mui/icons-material';
+import React, { FunctionComponent, useState } from 'react';
+import { useIntl } from 'react-intl';
+import EditableTableCell from './table-cell';
+
+interface TableRowComponentProps {
+    arrayFormName: string;
+    columnsDefinition: any;
+    row: any;
+    index: number;
+    handleDeleteButton: (index: number) => void;
+}
+
+const TableRowComponent: FunctionComponent<TableRowComponentProps> = ({
+    arrayFormName,
+    columnsDefinition,
+    row,
+    index,
+    handleDeleteButton,
+}) => {
+    const [hover, setHover] = useState(true);
+    const intl = useIntl();
+
+    return (
+        <TableRow
+            key={row.id}
+            onMouseEnter={() => setHover(true)}
+            onMouseLeave={() => setHover(false)}
+        >
+            {/* text={<RawReadOnlyInput name={`${arrayFormName}.${index}`} />}*/}
+            {columnsDefinition.map((column: any) =>
+                EditableTableCell(arrayFormName, index, column)
+            )}
+            <TableCell>
+                <Tooltip
+                    title={intl.formatMessage({
+                        id: 'DeleteRows',
+                    })}
+                    placement="top"
+                >
+                    <span hidden={!hover}>
```

Comment:
you can use css directly
```suggestion
                    <span style={{ ':hover': { display: none } }}
```

Reply:
In this specific case,I opted for managing the hover effect through usestate that aligns with React's principles and provides the flexibility and control necessary for a dynamic and interactive user interface.
If there are specific reasons for preferring the CSS approach in this context, I'm open to discussing them further and making adjustments if necessary.

Reply:
just because it necessitate to use js and got through react (events, callback, re-render) to react to the mouse, instead of just letting the navigator manage it itself with better performances